driver-core/src/test/java/com/datastax/driver/core/exceptions/ExceptionsScassandraTest.java

```diff
@@ -78,7 +78,7 @@ public void should_throw_proper_read_timeout_exception() {
     } catch (ReadTimeoutException e) {
       assertThat(e.getMessage())
           .isEqualTo(
-              "Cassandra timeout during read query at consistency LOCAL_ONE (1 responses were required but only 0 replica responded)");
+              "Cassandra timeout during read query at consistency LOCAL_ONE (1 responses were required but only 0 replica responded). In case this was generated during read repair, the consistency level is not representative of the actual consistency.");
```

Comment:
Note that this was changed relatively recently: see https://github.com/datastax/java-driver/pull/1568